### ISSUE TITLE
[SANDBOX PROJECT ONBOARDING] sealer

### ISSUE THREAD

**Issue by amye** [cncf/toc#831]
Welcome to CNCF Project Onboarding!
This is an issue created to help onboard your project into the CNCF after the TOC has voted to accept your project.
We would like to complete onboarding within one month of acceptance.

From the project side, please ensure that you:

- [ ] Understand the project proposal process and reqs: <https://github.com/cncf/toc/blob/main/process/project_proposals.adoc#introduction>
- [ ] Understand the services available for your project at CNCF <https://www.cncf.io/services-for-projects/>
- [ ] Ensure your project meets the CNCF IP Policy: <https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy>
- [ ] Review the online programs guidelines: <https://github.com/cncf/foundation/blob/master/online-programs-guidelines.md>
- [ ] Understand the trademark guidelines: <https://www.linuxfoundation.org/en/trademark-usage/>
- [ ] Understand the license allowlist: <https://github.com/cncf/foundation/blob/master/allowed-third-party-license-policy.md#approved-licenses-for-allowlist>
- [ ] Is your project working on written, open governance? see <https://contribute.cncf.io/maintainers/governance/>
- [ ] Slack: Are your slack channels migrated to the Kubernetes or CNCF Slack? (see <https://slack.com/help/articles/217872578-Import-data-from-one-Slack-workspace-to-another> for more details)
- [ ] Is your project in its own separate neutral github organization?
- [ ] Submitted a Pull request to add your project as a sandbox project to <https://landscape.cncf.io>
- [ ] Create maintainer list + add to aggregated <https://maintainers.cncf.io> list by submitting a PR to it
- [ ] Have added your project to <https://github.com/cncf/contribute>
- [ ] Artwork: Submit a pull request to <https://github.com/cncf/artwork> with your artwork
- [ ] Domain: transfer domain to the CNCF - <https://jira.linuxfoundation.org/plugins/servlet/theme/portal/2/create/63>

Things that CNCF will need from the project:

- [ ] Provide emails for the maintainers added to <https://maintainers.cncf.io> in order to get access to the maintainers mailing list and ServiceDesk
- [ ] Trademarks: transfer any trademark and logo mark assets over to the LF - <https://github.com/cncf/foundation/tree/master/agreements> has agreements
- [ ] GitHub: ensure 'thelinuxfoundation' and 'caniszczyk' are added as initial org owners, this helps us make sure we have continuity of GH ownership
- [ ] GitHub: ensure [DCO](https://github.com/apps/dco) or [CLA](https://github.com/cncf/cla) are enabled for all GitHub repositories of the project
- [ ] GitHub: ensure that hat the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) (or your adopted version of it) are explicitly referenced at the project's README on GitHub
- [ ] Website: ensure LF footer is there and [website guidelines](https://github.com/cncf/foundation/blob/master/website-guidelines.md) followed (if your project doesn't have a dedicated website, please adopt those guidelines to the README file of your project on GitHub).
- [ ] Website: Analytics transferred to amye@linuxfoundation.org
- [ ] CII: Start on a CII best practices badge <https://bestpractices.coreinfrastructure.org/en>

Things that the CNCF will do or help the project to do:

- [ ] Devstats: add to devstats <https://devstats.cncf.io/>
- [ ] Marketing: update relevant intro + slide decks
- [ ] Events: update CFP + Registration + CFP Area forms
- [ ] ServiceDesk: confirm maintainers have read <https://www.cncf.io/services-for-projects/>
- [ ] CNCF Welcome Email Sent to confirm maintainer list access, welcome email has monthly project sync details
- [ ] Create space for meetings/events on <https://community.cncf.io>, e.g., <https://community.cncf.io/pravega-community/> - (<https://github.com/cncf/communitygroups/blob/main/README.md#cncf-projects>)
- [ ] Adopt a license scanning tool, like [FOSSA](https://fossa.com/) or [Snyk](https://snyk.io/)

**Comment by amye** [comment]
@fanux - We're in here! It's great to see you all! 
First things: landscape and maintainers.cncf.io is a great place to start!

**Comment by fanux** [comment]
I got it~

**Comment by lukaszgryglicki** [comment]
Test [DevStats](https://sealer.teststats.cncf.io) page added.

**Comment by lukaszgryglicki** [comment]
Prod [DevStats](https://sealer.devstats.cncf.io) page added, ETA for adding new project to [All CNCF](https://all.devstats.cncf.io) is tomorrow (same with adding it to `Project(s) Health).

**Comment by fanux** [comment]
Maintainers https://github.com/alibaba/sealer/blob/main/MAINTAINERS.md https://github.com/cncf/foundation/pull/335 added.

**Comment by lukaszgryglicki** [comment]
New project added to [All CNCF](https://all.devstats.cncf.io/d/1/activity-repository-groups?orgId=1&var-period=h24&var-repogroups=%22OpenFunction%22&var-repogroups=%22Teller%22&var-repogroups=%22sealer%22) instance and [Projects Health](https://all.devstats.cncf.io/d/53/projects-health-table?orgId=1).

**Comment by fanux** [comment]
'thelinuxfoundation' and 'caniszczyk' are added as initial org owners: https://github.com/sealerio

**Comment by fanux** [comment]
How to enable [CLA](https://github.com/cncf/cla) for a GitHub repositories? @amye DCO already enabled.

**Comment by amye** [comment]
DCO is fine, I'll mark that off. We only need one, not both.

**Comment by fanux** [comment]
@amye add maintainers done: https://github.com/cncf/foundation/pull/335

**Comment by fanux** [comment]
contribute landscape logo done: https://github.com/cncf/landscape/pull/2572

**Comment by fanux** [comment]
@amye  How to transfer analytics to you? And we already done some work, could you mark them off ?

**Comment by fanux** [comment]
@amye how to transfer Analytics to you? And we have already done some work, could you mark them off?

**Comment by amye** [comment]
> 

Please invite projects@cncf.io to your Google Analytics account.

**Comment by amye** [comment]
Checking back in here, I don't yet have emails for you all! 
Drop an email to cncf-maintainer-changes@cncf.io and we'll get you taken care of!

**Comment by allencloud** [comment]
@amye 

I have sent an email including sealer maintainers' email address to [cncf-maintainer-changes@cncf.io](mailto:cncf-maintainer-changes@cncf.io), and all those email addresses could be also found at https://github.com/sealerio/sealer/blob/main/MAINTAINERS.md

- [] Provide emails for the maintainers added to https://maintainers.cncf.io/ in order to get access to the maintainers mailing list and ServiceDesk
- [ ] GitHub: ensure that hat the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) (or your adopted version of it) are explicitly referenced at the project's README on GitHub. see https://github.com/sealerio/sealer#code-of-conduct
- [ ]  CII: Start on a CII best practices badge https://bestpractices.coreinfrastructure.org/en. Done, see https://bestpractices.coreinfrastructure.org/en/projects/5205
- [ ] Adopt a license scanning tool, like [FOSSA](https://fossa.com/) or [Snyk](https://snyk.io/). Done, see https://app.fossa.com/projects/git%2Bgithub.com%2Fsealerio%2Fsealer/refs/branch/main/304fda7ce4801f97ed992cc8cde375ae2f1539eb/preview

All the above are done. Please help to check and mark. Thanks.

**Comment by amye** [comment]
Marking off as done, invites are out for servicedesk and maintainers lists!

**Comment by allencloud** [comment]
> Marking off as done, invites are out for servicedesk and maintainers lists!

We have received the invitation of servicedesk and maintainers lists.

**Comment by fanux** [comment]
- [x]  Trademarks: transfer any trademark and logo mark assets over to the LF - https://github.com/cncf/foundation/tree/master/agreements has agreements   
- [x] Website: ensure LF footer is there and [website guidelines](https://github.com/cncf/foundation/blob/master/website-guidelines.md) followed (if your project doesn't have a dedicated website, please adopt those guidelines to the README file of your project on GitHub).

@amye These two have been completed

**Comment by amye** [comment]
Trademarks: transfer any trademark and logo mark assets over to the LF - https://github.com/cncf/foundation/tree/master/agreements has agreements -- we'll need a fully executed copy from both sides. I don't have anything in the project onboarding inbox? project-onboarding@cncf.io is a great place to put that.

**Comment by starnop** [comment]
@Cmierly Maybe something went wrong somewhere and we re-signed the [agreement](https://github.com/cncf/foundation/tree/master/agreements). Before the official agreement is approved, I would like to confirm with you whether there is any problem with the content of the agreement. 

So is it convenient for you to provide a suitable way? I will send the draft agreement to you for preliminary review. THX. 😄

**Comment by starnop** [comment]
@Cmierly Maybe something went wrong somewhere and we re-signed the [agreement](https://github.com/cncf/foundation/tree/master/agreements). Before the official agreement is approved, I would like to confirm with you whether there is any problem with the content of the agreement. 

So is it convenient for you to provide a suitable way? I will send the draft agreement to you for preliminary review. THX. 😄

**Comment by amye** [comment]
> @Cmierly Maybe something went wrong somewhere and we re-signed the [agreement](https://github.com/cncf/foundation/tree/master/agreements). Before the official agreement is approved, I would like to confirm with you whether there is any problem with the content of the agreement.
> 
> So is it convenient for you to provide a suitable way? I will send the draft agreement to you for preliminary review. THX. 😄

Not sure what the issue is? I don't have a signed copy of either registered or non-registered.

**Comment by starnop** [comment]
@amye The agreement needs to be approved before it can be used, and this approval process is not easy. Therefore, before going through the approval process, we want to confirm whether there is a problem with the content.

I wonder if it is appropriate for me to send a draft version to [project-onboarding@cncf.io](mailto:project-onboarding@cncf.io)

**Comment by amye** [comment]
> @amye The agreement needs to be approved before it can be used, and this approval process is not easy. Therefore, before going through the approval process, we want to confirm whether there is a problem with the content.
> 
> I wonder if it is appropriate for me to send a draft version to [project-onboarding@cncf.io](mailto:project-onboarding@cncf.io) and look forward to confirmation feedback on the agreement content

We only have two agreements - non-registered trademark and registered trademarks. If you want something different, we'll need to work with Legal on our end.

**Comment by starnop** [comment]
@amye We have not applied for any trademark regarding sealer even if for word mark, design, and logo marks. So we couldn't sign the agreement as assignor. I wonder if we have any good suggestions?


![image](https://github.com/cncf/toc/assets/31209634/985a5a12-0ab6-4b46-8f25-648064b59389)

**Comment by starnop** [comment]
@amye Now, we can only sign the aggrement（no reg trademarks) as the owner of the SEALER domain name. And I have sent a draft version to [project-onboarding@cncf.io](mailto:project-onboarding@cncf.io), please help to confirm that whether the content meets the requirements.

**Comment by amye** [comment]
Responded with more details, but I think we can take trademarks offline from here.
Happy to help with other onboarding tasks!

**Comment by krook** [comment]
[Added to LFX Insights](https://insights.lfx.linuxfoundation.org/foundation/cncf/overview?project=sealer)

- [Insights documentation](https://docs.linuxfoundation.org/lfx/insights/v3-beta-version-current)
- [Insights support](https://jira.linuxfoundation.org/plugins/servlet/desk/portal/4?requestGroup=54)
- [Insights feedback](https://community.lfx.dev/t/lfx-insights-v3-feedback/2430)
